extensions/amp-story/1.0/amp-story.js

```diff
@@ -764,6 +765,8 @@ export class AmpStory extends AMP.BaseElement {
         this.element.querySelector('amp-story-page'),
         'Story must have at least one page.');
 
+    // TODO(#20128): initialPageId will no longer be necessary here once
+    // branching experiment is live.
     const initialPageId = this.getHistoryState_(HistoryStates.PAGE_ID) ||
```

Comment:
Can we call `const initialPageId = this.getInitialPageId_(firstPageEl);` directly here, and do the experiment check in that method? This way, we could remove that block before the `switchTo_`, and keep the code like it was before.

Reply:
I believe that `getInitialPageId_` needs to be called later because at this point the pages_ array hasn't been initialized yet. 

How do feel about getting rid of the assignment of `initialPageId` entirely, still moving the experiment check to `getInitialPageID_`, and calling it in switchTo_ directly as the first argument.